sdks/android/Makefile

```diff
@@ -42,9 +45,11 @@ assets/mconfig: machine.config assets
 stage-sdk: $(foreach lib,$(MONO_LIBS),jni/armeabi-v7a/$(lib)) $(foreach lib,$(MONO_LIBS),jni/arm64-v8a/$(lib)) \
 	$(foreach lib,$(MONO_LIBS),jni/x86/$(lib)) $(foreach lib,$(MONO_LIBS),jni/x86_64/$(lib)) assets/mconfig
 
+#This is hilarious, but we can't clean unless we got the 2 property files around
```

Comment:
Won't `rm -f` work?

Reply:
It would, but that would require us to do ant's job which, at that point, clear could just be a `git clean .`.